python_scripts/ExploreData.py

```diff
@@ -57,14 +57,15 @@ def HandleNanData(dataFrame, categoricalFeatureList):
 def HandleCategoricalData(dataFrame, categoricalFeatureList):
 	
 	for catfeat in categoricalFeatureList:
-		dataFrame = pd.concat([dataFrame, pd.get_dummies(dataFrame[catfeat],      sparse=True)])
+		print catfeat + " includes "+str(len(dataFrame[catfeat].unique()))+" different categorical values"
+		dataFrame = pd.concat([dataFrame, pd.get_dummies(dataFrame[catfeat],      sparse=False)],axis=1)
 		dataFrame = dataFrame.drop([catfeat],axis=1)
 	
 	return dataFrame		
 
 def SplitTrainValData(data, label, val_size=0.2):
```

Comment:
This pulls random 20% for training? Could we be more thorough with a k-folds approach?